gluten-substrait/src/main/scala/org/apache/gluten/expression/DateTimeExpressionsTransformer.scala

```diff
@@ -102,3 +103,18 @@ object DateTimeExpressionsTransformer {
     scala.reflect.classTag[Second].runtimeClass -> "SECOND"
   )
 }
+
+case class ToUnixTimestampTransformer(
```

Comment:
Consider moving this transformer to 'incubator-gluten/backends-velox/src/main/scala/org/apache/gluten/expression/ExpressionTransformer.scala' because it's only used by Velox.